### PR TITLE
Add pci dss

### PR DESCRIPTION
**Pull request by tomschr**
TODO:

* Correct the respective docserv config file.

### CODE REVIEW

File: xml/art_pci-dss.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ <titleabbrev>&sle_pcidss_guide;</titleabbrev>
+ <info>
+  <productname>&productnamex;</productname>
+  <productnumber>&productnumberx;</productnumber>
+  <date><?dbtimestamp format="B d, Y" ?></date>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+    <dm:component>Documentation</dm:component>
+    <dm:product>SUSE Linux Enterprise Server 15 SP4</dm:product>
```
Comment (cjschroder) [inline xml/art_pci-dss.xml:28]: Isn't this article supposed to be unversioned?

File: xml/art_pci-dss.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ <xi:include xpointer="element(/1/3)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/4)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/5)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/6)" href="security_pcidss.xml"/>
+
+ <!-- sections: -->
+ <xi:include xpointer="element(/1/7)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/8)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/9)" href="security_pcidss.xml"/>
+
```
Comment (cjschroder) [inline xml/art_pci-dss.xml:47]: not sure about all of these xpointers. I'm finding a few typos and tagging errors, I will do a thorough proofread in a separate PR.

File: xml/common_gfdl1.2_i.xml [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+  of the three, merge those two alternatives to suit the situation.
+ </para>
+
+ <para>
+  If your document contains nontrivial examples of program code, we recommend
+  releasing these examples in parallel under your choice of free software
+  license, such as the GNU General Public License, to permit their use in free
+  software.
+ </para>
+</sect1>
```
Comment (cjschroder) [inline xml/common_gfdl1.2_i.xml:544]: This license text is too short. I will add more. 
🙄

**Review by cjschroder — APPROVED** [review]
This looks good, thank you for your hard work!

File: xml/art_pci-dss.xml (hunk 1 above)
Comment (tomschr) [reply xml/art_pci-dss.xml:28]: That's a good question. Well, should be, right? However, I just copied it from doc-sle. :slightly_smiling_face:

Comment (tomschr) [reply xml/art_pci-dss.xml:28]: This part is needed to fill our bugtracker. So we can leave it as is for the time being.

**Pull request merged**